content/typescript/use-enums-when-needed.md

```diff
@@ -0,0 +1,47 @@
+---
+title: define enums for variables with already known multiple possible values
+---
+
+# Problem
+
+When we're working with variables, sometimes you'd need to have variables with already known multiple possible values, that will always be the same in your application (think about having a ticket status variable, `CLOSED`, `OPEN`, `WAITING RESPONSE`)
+
+# Solution
+
+Typescript allows us to define enumerators, which are variables with multiple defined possible values. By using enumerables TS will be able to infer and autocomplete the possible values of a variable inside your code.
```

Comment:
enumerators -> enums
TS -> TypeScript
`autocomplete the possible values` -> `autocomplete possible values`